README.md

```diff
@@ -2,6 +2,85 @@
 
 Bot for Scratch Addons Discord server
 
-[Cloud hosted on opeNode.io](https://www.openode.io/)
+Cloud hosted on [opeNode.io](https://www.openode.io/)
+
+## Features
+### General
+---
+### PotatoBoard
```

Comment:
Separate into two words?

Reply:
no it's one word but dont capitalize board

Reply:
looks better with capitalization imo but fine